src/components/Navbar.js

```diff
@@ -0,0 +1,172 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import {
+  AppBar,
+  Box,
+  Toolbar,
+  IconButton,
+  Typography,
+  Menu,
+  MenuItem,
+  Container,
+  Avatar,
+  Button,
+  Tooltip,
+} from "@mui/material";
+import MenuIcon from "@mui/icons-material/Menu";
+import AdbIcon from "@mui/icons-material/Adb";
+
+// const pages = ['Products', 'Pricing', 'Blog'];
+const pages = ["Home", "Menu", "Contact"];
```

Comment:
Clicking on the button doesn't update the URL, so it isn't attempting the change the page.

Reply:
I've removed this import and its lines of code.

Reply:
Added styles to links.